keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -313,8 +307,9 @@ No new integration tests for kubelet are planned.
 
 #### Beta
 
-- [ ] Gather feedback from consumers of the new policy option.
-- [ ] Verify no major bugs reported in the previous cycle.
+- [X] Gather feedback from consumers of the new policy option.
+- [X] Verify no major bugs reported in the previous cycle.
+- [X] Ensure proper e2e tests are in place.
```

Comment:
According to our [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#e2e-tests) for beta promotion we require links to testgrid where these tests are being run. Based on the previous description it was mentioned several tests should have been added, were they? Because based on this description it doesn't look like so? Can we get them added asap?

Reply:
Unit tests are added 1.32; e2e tests are to be added after this KEP is approved.
Are you asking the e2e tests be added before this KEP is approved? chicken and egg?